diff -up tornado-4.3/setup.py.requires tornado-4.3/setup.py
--- tornado-4.3/setup.py.requires	2016-09-15 21:31:05.188426632 -0600
+++ tornado-4.3/setup.py	2016-09-15 21:49:19.093465027 -0600
@@ -124,8 +124,6 @@ if setuptools is not None:
     if sys.version_info < (2, 7):
         # Only needed indirectly, for singledispatch.
         install_requires.append('ordereddict')
-    if sys.version_info < (3, 2):
-        install_requires.append('backports.ssl_match_hostname')
     if sys.version_info < (3, 4):
         install_requires.append('singledispatch')
     if sys.version_info < (3, 5):
