diff --git a/tornado/test/runtests.py b/tornado/test/runtests.py
index 526ae5d..36a841f 100644
--- a/tornado/test/runtests.py
+++ b/tornado/test/runtests.py
@@ -124,6 +124,9 @@ def main():
     # Silence the warning until we can drop 3.5.[01].
     warnings.filterwarnings("ignore", category=PendingDeprecationWarning,
                             message=".*legacy __aiter__ protocol")
+    # 3.5.2's PendingDeprecationWarning became a DeprecationWarning in 3.6.
+    warnings.filterwarnings("ignore", category=DeprecationWarning,
+                            message=".*legacy __aiter__ protocol")
 
     logging.getLogger("tornado.access").setLevel(logging.CRITICAL)
 
